compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -44,4 +44,15 @@ def Cleanup :
       "mlir::iree_compiler::AMDAIE::createCleanupPass()";
 }
 
+def AMDAIEBufferizeToAllocation :
+    Pass<"iree-amdaie-bufferize-to-allocation", "IREE::HAL::ExecutableVariantOp"> {
+  let summary = "Pass to bufferizes the targeted operation and materializes the result in a new allocation.";
+  let constructor =
+      "mlir::iree_compiler::AMDAIE::createAMDAIEBufferizeToAllocationPass()";
+  let options = [
+    Option<"memorySpace", "memory-space", "int64_t", /*default=*/"1",
+      "Set the memory space">
+  ];
+}
+
```

Comment:
Since we are removing the "bufferize" from `--iree-amdaie-pad-and-bufferize` - we should rename the latter as just `--iree-amdaie-pad` (or maybe some other name CC: @MaheshRavishankar ).

This renaming should even reflect on the DEBUG_TYPE of the original pass.

Reply:
yeah, rename the passes `--iree-amdaie-pad` and `--iree-amdaie-bufferize`.

Reply:
Done